integration_test/third_party_apps_test/main_test.go

```diff
@@ -131,7 +131,7 @@ func distroFolder(platform string) (string, error) {
 }
 
 func readFileFromScriptsDir(scriptPath string) ([]byte, error) {
```

Comment:
this function is now silly and we should inline its contents into each callsite.